frontend/src/metabase/sharing/components/PulsesListSidebar.jsx

```diff
@@ -103,11 +103,14 @@ function buildRecipientText(pulse) {
   const {
     channels: [firstChannel],
   } = pulse;
-  if (firstChannel.channel_type !== "email") {
+
+  const { channel_type, recipients } = firstChannel;
+
+  if (channel_type !== "email" || _.isEmpty(recipients)) {
```

Comment:
Although it shouldn't be possible any more to intentionally create a subscription without recipients, we still need to guard against that case.